addons/resource.language.en_gb/resources/strings.po

```diff
@@ -15840,25 +15840,30 @@ msgstr ""
 
 #: xbmc/addons/GUIDialogAddonInfo.cpp
```

Comment:
MInd adding verbal descriptions to the new/changed strings? They are not self-explaining. Please help the translaters.

Reply:
sure, done